bats/tests/helpers/vm.bash

```diff
@@ -39,7 +39,71 @@ flush_iptables() {
     wsl sudo iptables -X
 }
 
+# Helper to eject all existing ramdisk instances on macOS
+macos_eject_ramdisk() {
+    local mount="$1"
+    run hdiutil info -plist
+    assert_success
+    # shellcheck disable=2154 # $output set by `run`
+    run plutil -convert json -o - - <<<"$output"
+    assert_success
+    # shellcheck disable=2016 # $mount is interpreted by jq, not shell.
+    run jq_output --arg mount "$mount" \
+        '.images[]."system-entities"[] | select(."mount-point" == $mount) | ."dev-entry"'
+    assert_success
+    if [[ -z $output ]]; then
+        return
+    fi
+    # We don't need to worry about splitting here, it's all /dev/disk*
+    # However, we do need to ensure $output isn't clobbered.
+    # shellcheck disable=2206
+    local disks=($output)
+    local disk
+    for disk in "${disks[@]}"; do
+        umount "$disk" 2>&1 | sed 's@^@# @' >&3 || :
+    done
+    for disk in "${disks[@]}"; do
+        hdiutil eject "$disk" 2>&1 | sed 's@^@# @' >&3 || :
+    done
+}
+
+# Set up the use of a ramdisk for application data, to make things faster.
+setup_ramdisk() {
+    if ! using_ramdisk; then
+        return
+    fi
+    if is_macos; then
+        # Try to eject the disk, if it already exists.
+        macos_eject_ramdisk "$LIMA_HOME"
+
+        local ramdisk_size="${RD_RAMDISK_SIZE}"
+        if [[ $ramdisk_size == "auto" ]]; then
+            ramdisk_size="${RD_FILE_RAMDISK_SIZE:-5}" # default to 5GB
+        fi
+        local sectors=$((ramdisk_size * 1024 * 1024 * 1024 / 512)) # Size, in sectors.
+        # hdiutil space-pads the output; strip it.
+        disk="$(hdiutil attach -nomount "ram://$sectors" | xargs echo)"
+        newfs_hfs "$disk"
```

Comment:
Can we give the volume a label, so it doesn't show as "untitled" in my volume list:

```suggestion
        newfs_hfs -v "Rancher Desktop BATS" "$disk"
```